app/assets/stylesheets/button.sass

```diff
@@ -0,0 +1,3 @@
+
+.add_to_cart
```

Comment:
@Robbie-Smith we added an .add_to_cart class, so that should come in handy when selecting it on the tests.